### PR TITLE
feature - view tutorial screen

### PR DESCRIPTION
**Pull request by at-cuongdoan**

### CODE REVIEW

File: MyApp/View/Tutorial/TutorialViewController.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        super.viewDidLoad()
+
+        scrollView.delegate = self
+    }
+
+    @IBAction func pushNext(_ sender: UIButton?) {
+        if pageControl.currentPage < viewModel.images.count - 1 {
+            let x = CGFloat(scrollView.contentOffset.x + scrollView.frame.width)
+            scrollView.setContentOffset(CGPoint(x: x, y: 0), animated: true)
+            pageControl.currentPage = pageControl.currentPage + 1
```
Comment (at-daiho) [inline MyApp/View/Tutorial/TutorialViewController.swift]: ERROR - Prefer shorthand operators (+=, -=, *=, /=) over doing the operation and assigning.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 0

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Tutorial/TutorialViewController.swift [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    var viewModel = TutorialViewModel()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        scrollView.delegate = self
+    }
+
+    @IBAction func pushNext(_ sender: UIButton?) {
```
Comment (at-hieutran) [inline MyApp/View/Tutorial/TutorialViewController.swift]: Thêm Mark vô

File: MyApp/View/Tutorial/TutorialViewController.swift [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            AppDelegate.changeViewLogin()
+        }
+    }
+
+    @IBAction func pushSkip(_ sender: UIButton?) {
+        AppDelegate.changeViewLogin()
+    }
+}
+
+extension TutorialViewController: UIScrollViewDelegate {
```
Comment (at-hieutran) [inline MyApp/View/Tutorial/TutorialViewController.swift]: Đây mark nữa

File: MyApp/View/Tutorial/TutorialViewController.swift [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import MVVM
+
+class TutorialViewController: UIViewController, MVVM.View {
+    @IBOutlet weak var scrollView: UIScrollView!
+    @IBOutlet weak var pageControl: UIPageControl!
+    @IBOutlet weak var nextButton: UIButton!
+
+    var viewModel = TutorialViewModel()
+
+    override func viewDidLoad() {
```
Comment (at-hieutran) [inline MyApp/View/Tutorial/TutorialViewController.swift]: Thêm Mark vô

File: MyApp/View/Tutorial/TutorialViewController.swift [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+//
+//  Created by Cuong Doan M. on 11/29/17.
+//  Copyright © 2017 Cuong Doan M. All rights reserved.
+//
+
+import UIKit
+import MVVM
+
+class TutorialViewController: UIViewController, MVVM.View {
+    @IBOutlet weak var scrollView: UIScrollView!
```
Comment (at-hieutran) [inline MyApp/View/Tutorial/TutorialViewController.swift]: Thêm Mark vô

File: MyApp/View/Tutorial/TutorialViewController.swift [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+//  Tutorial
+//
+//  Created by Cuong Doan M. on 11/29/17.
+//  Copyright © 2017 Cuong Doan M. All rights reserved.
+//
+
+import UIKit
+import MVVM
+
+class TutorialViewController: UIViewController, MVVM.View {
```
Comment (at-hieutran) [inline MyApp/View/Tutorial/TutorialViewController.swift]: Thiếu final

**Pull request merged**